roles/cs.mageops-cli-user/templates/bashrc.d/debug-tunnel.bash

```diff
@@ -0,0 +1,26 @@
+{% if php_fpm_debug_pool_xdebug_remote_port | default(false, true) and mageops_debug_token | default(false, true) %}
+cat<<ENDMSG
+
+  ------------------  $(echo -en "\033[1;36mXDebug Remote Host\033[0m")  ------------------
+
+  $(echo -en "\033[1;36mDebugging token:\033[0m") $(echo -en "\033[0;35m{{ mageops_debug_token }}\033[0m")
+  $(echo -en "\033[1;36mXDebug port:\033[0m") $(echo -en "\033[0;35m{{ php_fpm_debug_pool_xdebug_remote_port }}\033[0m")
+
+  ---------------------------------------------------------
+
+  1. Set up a tunnel from your $(echo -en "\033[1mlocal computer\033[0m"):
+
+  $(echo -en "\033[0;35m")ssh -N \\
+    -o ExitOnForwardFailure=yes \\
+    -o GatewayPorts=yes \\
+    -R {{ php_fpm_debug_pool_xdebug_remote_port }}:localhost:{{ php_fpm_debug_pool_xdebug_remote_port }} \\
+      $(whoami)@$(curl -sL https://api.ipify.org)$(echo -en "\033[0m")
```

Comment:
Please extract this URL to a setting.
Please use http://169.254.169.254/latest/meta-data/public-ipv4 for AWS

Reply:
Fixed as agreed.